src/main/java/com/example/account/service/command/impl/CommandServiceImpl.java

```diff
@@ -34,4 +36,21 @@ public void createAccount(CreateAccountCommand command) {
                          }).subscribe();
     }
 
+    @Override
+    public void update(UpdateAccountCommand command) {
+        AccountAggregate accountAggregate = AccountAggregate.builder()
+                                                            .status(command.getStatus())
+                                                            .name(command.getName())
+                                                            .balance(command.getBalance())
+                                                            .reserved(command.getReserved())
+                                                            .auctionId(command.getAuctionId())
+                                                            .build();
+        accountRepository.save(accountAggregate)
+                         .doOnNext(accountUpdated -> {
+                             AccountUpdatedEvent event = new AccountUpdatedEvent();
+                             event.apply(accountAggregate);
+                             eventHandler.handle(event);
+                         }).subscribe();
```

Comment:
don't break the reactive pipeline, don't use `subscribe` in reactive context

Reply:
@amerkul `subscribeOn` can help

Reply:
Or just return a Mono object via controller.